diff --git a/node_modules/react-native/Libraries/Performance/Systrace.js b/node_modules/react-native/Libraries/Performance/Systrace.js
index e1db39b..7e85aab 100644
--- a/node_modules/react-native/Libraries/Performance/Systrace.js
+++ b/node_modules/react-native/Libraries/Performance/Systrace.js
@@ -202,7 +202,7 @@ if (__DEV__) {
   // other files. Therefore, calls to `require('moduleId')` are not replaced
   // with numeric IDs
   // TODO(davidaurelio) Scan polyfills for dependencies, too (t9759686)
-  (require: any).Systrace = Systrace;
+  // (require: any).Systrace = Systrace;
 }
 
 module.exports = Systrace;
